src/scss/components/_picture.scss

```diff
@@ -29,7 +29,6 @@
   &__image {
     background-color: get-color(air);
     display: block;
-    height: auto;
```

Comment:
@iondrimba removing this `height: auto` does not affect picture images. Removing here, it's ok! :)

PS: We need use width/height always in all images with this removal. This is a best practice, indeed.